src/backend/utils/storage.js

```diff
@@ -76,6 +77,10 @@ module.exports = {
       .exec();
   },
 
+  flagFeed: (id) => redis.smove(feedsKey, flaggedFeedsKey, id),
+
+  unflagFeed: (id) => redis.smove(flaggedFeedsKey, feedsKey, id),
```

Comment:
I like the idea of doing `smove`, but I have a question: what other functions touch `feedsKey` that need to be updated to also know about `flaggedFeedsKey`?  For example, if we delete a feed, it could be in one or the other, so we'd need to do it on both sets.  Are there other cases?

We'll also need a way to get all flagged feeds.